### ISSUE TITLE
[terra-progress-bar] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-core#3051]
# Bug Report

The following themeable variables are missing (we should theme them even though they're dev site only for example purposes):

**clinical-lowlight-theme**

--terra-progress-bar-2-colors-example-1-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 48, column: 30)
--terra-progress-bar-2-colors-example-1-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 51, column: 18)
--terra-progress-bar-2-colors-example-2-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 57, column: 30)
--terra-progress-bar-2-colors-example-2-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 60, column: 18)
--terra-progress-bar-2-colors-example-3-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 66, column: 30)
--terra-progress-bar-2-colors-example-3-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 69, column: 18)
--terra-progress-bar-customized-example-1-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 78, column: 32)
--terra-progress-bar-customized-example-1-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 92, column: 8)
--terra-progress-bar-customized-example-1-empty-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 101, column: 20)
--terra-progress-bar-customized-example-1-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 91, column: 8)
--terra-progress-bar-customized-example-2-backup-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 115, column: 8)
--terra-progress-bar-customized-example-2-backup-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 132, column: 8)
--terra-progress-bar-customized-example-2-bar-background - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 114, column: 8)
--terra-progress-bar-customized-example-2-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 133, column: 8)
--terra-progress-bar-customized-example-2-empty-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 142, column: 20)
--terra-progress-bar-customized-example-2-value-background - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 131, column: 8)
--terra-progress-bar-example-1-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 28, column: 18)
--terra-progress-bar-example-2-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 35, column: 18)
--terra-progress-bar-example-3-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 42, column: 18)

**orion-fusion-theme**

--terra-progress-bar-2-colors-example-1-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 48, column: 30)
--terra-progress-bar-2-colors-example-1-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 51, column: 18)
--terra-progress-bar-2-colors-example-2-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 57, column: 30)
--terra-progress-bar-2-colors-example-2-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 60, column: 18)
--terra-progress-bar-2-colors-example-3-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 66, column: 30)
--terra-progress-bar-2-colors-example-3-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 69, column: 18)
--terra-progress-bar-customized-example-1-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 78, column: 32)
--terra-progress-bar-customized-example-1-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 92, column: 8)
--terra-progress-bar-customized-example-1-empty-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 101, column: 20)
--terra-progress-bar-customized-example-1-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 91, column: 8)
--terra-progress-bar-customized-example-2-backup-bar-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 115, column: 8)
--terra-progress-bar-customized-example-2-backup-value-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 132, column: 8)
--terra-progress-bar-customized-example-2-bar-background - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 114, column: 8)
--terra-progress-bar-customized-example-2-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 133, column: 8)
--terra-progress-bar-customized-example-2-empty-border-right - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 142, column: 20)
--terra-progress-bar-customized-example-2-value-background - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 131, column: 8)
--terra-progress-bar-example-1-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 28, column: 18)
--terra-progress-bar-example-2-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 35, column: 18)
--terra-progress-bar-example-3-background-color - packages/terra-progress-bar/lib/terra-dev-site/doc/example/colors.module.scss (line: 42, column: 18)

**Comment by neilpfeiffer** [comment]
For the Lowlight theme:
```
:local {
  .clinical-lowlight-theme {
    /* ...existing styles here... */

    // Used for docsite custom color examples only 
    --terra-progress-bar-example-1-background-color: #fdb06c ;
    --terra-progress-bar-example-2-background-color: #427c1f;
    --terra-progress-bar-example-3-background-color: #f94e4e;
    --terra-progress-bar-2-colors-example-1-bar-background-color: #222a2e;
    --terra-progress-bar-2-colors-example-1-value-background-color: #004c76;
    --terra-progress-bar-2-colors-example-2-bar-background-color: #222a2e;
    --terra-progress-bar-2-colors-example-2-value-background-color: #004c76;
    --terra-progress-bar-2-colors-example-3-bar-background-color: #222a2e;
    --terra-progress-bar-2-colors-example-3-value-background-color: #004c76;
    --terra-progress-bar-customized-example-1-bar-background-color: #222a2e;
    --terra-progress-bar-customized-example-1-value-background-color: #427c1f;
    --terra-progress-bar-customized-example-1-border-right: 1px solid #fff;
    --terra-progress-bar-customized-example-1-empty-border-right: 0 solid #fff;
    --terra-progress-bar-customized-example-2-bar-background: linear-gradient(to left, rgb(255, 255, 0), rgb(0, 0, 255));
    --terra-progress-bar-customized-example-2-backup-bar-background-color: rgb(0, 0, 255);
    --terra-progress-bar-customized-example-2-value-background:linear-gradient(to left, rgb(255, 0, 0), rgb(0, 128, 0));
    --terra-progress-bar-customized-example-2-backup-value-background-color: rgb(255, 0, 0);
    --terra-progress-bar-customized-example-2-border-right: 3px solid rgb(0, 255, 0);
    --terra-progress-bar-customized-example-2-empty-border-right: 0 solid rgb(0, 255, 0);
  }
}
```

----

For the Fusion theme:
```
:local {
  .orion-fusion-theme {
    /* ...existing styles here... */

    // Used for docsite custom color examples only 
    --terra-progress-bar-example-1-background-color: #f00;
    --terra-progress-bar-example-2-background-color: #ffc20a;
    --terra-progress-bar-example-3-background-color: #008000;
    --terra-progress-bar-2-colors-example-1-bar-background-color: #f4f4f4;
    --terra-progress-bar-2-colors-example-1-value-background-color: #008000;
    --terra-progress-bar-2-colors-example-2-bar-background-color: #f4f4f4;
    --terra-progress-bar-2-colors-example-2-value-background-color: #008000;
    --terra-progress-bar-2-colors-example-3-bar-background-color: #f4f4f4;
    --terra-progress-bar-2-colors-example-3-value-background-color: #008000;
    --terra-progress-bar-customized-example-1-bar-background-color: #d3d4d5;
    --terra-progress-bar-customized-example-1-value-background-color: #8ccc62;
    --terra-progress-bar-customized-example-1-border-right: 1px solid #fff;
    --terra-progress-bar-customized-example-1-empty-border-right: 0px solid #fff;
    --terra-progress-bar-customized-example-2-bar-background: linear-gradient(to left, rgb(255, 255, 0), rgb(0, 0, 255));
    --terra-progress-bar-customized-example-2-backup-bar-background-color: rgb(0, 0, 255);
    --terra-progress-bar-customized-example-2-value-background:linear-gradient(to left, rgb(255, 0, 0), rgb(0, 128, 0));
    --terra-progress-bar-customized-example-2-backup-value-background-color: rgb(255, 0, 0);
    --terra-progress-bar-customized-example-2-border-right: 3px solid rgb(0, 255, 0);
    --terra-progress-bar-customized-example-2-empty-border-right: 0 solid rgb(0, 255, 0);
  }
}
```

**Comment by emilyrohrbough** [comment]
Resolved in https://github.com/cerner/terra-core/pull/3107
